src/robot2017/src/robot_exec.cpp

```diff
@@ -57,6 +59,7 @@ void RobotExec::autonomyReceived(const robot_msgs::Autonomy& cmd)
     {
         autonomyExec(cmd);
     }
+    publishMotors();
```

Comment:
Good for now but see if you can call this via a ROS timer so it does not require receiving input from autonomy